### DISCUSSION TITLE
在自己的开源项目中使用得意黑是否需要申请授权

### DISCUSSION THREAD

**Discussion by woshishabii** [atelier-anchor/smiley-sans#115] [Q&A] [answered]
在给班里整一个每日一言，想使用得以黑作为字体，不知道是否需要申请授权
以及得意黑应该以什么形式出现在项目中（ttf文件作为项目一部分还是在actions中下载）

**Comment by woshishabii** [comment]
放假时间有限可能等不到回复了，我先使用actions下载了

**Comment by stone-zeng** [comment, marked as answer]
并不需要，具体请看 [LICENSE](https://github.com/atelier-anchor/smiley-sans/blob/main/LICENSE)。

**Reply by stone-zeng** [reply]
（另外不建议重复提和 discussion 一样的 issue
